inc/classes/subscriber/third-party/plugins/Images/Webp/class-imagify-subscriber.php

```diff
@@ -346,7 +346,7 @@ public function is_serving_webp() {
 			// No Imagify, no webp.
 			return false;
 		}
-		return (bool) ( get_imagify_option( 'display_webp' ) || get_imagify_option( 'display_nextgen' ) );
+		return (bool) ( get_imagify_option( 'display_webp' ) || get_imagify_option( 'display_nextgen' ) || get_imagify_option( 'convert_to_avif' ) );
```

Comment:
I don't think we should have 
``` php
|| get_imagify_option( 'convert_to_avif' )
``` 
in this condition, as `convert_to_avif` is the option for enabling AVIF, but not displaying it. Which means if we enable AVIF, it will return true even if we dont' serve them through Imagify